source/Calamari.Common/Plumbing/Logging/RedactedValuesLogger.cs

```diff
@@ -17,8 +15,9 @@ public RedactedValuesLogger(ILog innerLogger)
             log = innerLogger;
         }
 
-        public void AddValueToRedact(string value, string replacement)
+        public void AddValueToRedact(string? value, string replacement)
```

Comment:
I don't like having a nullable value here - if there's a location which may pass a null value, can we do the check there instead?

Reply:
Fixed.